### PR TITLE
Fix/automate quick notes

### PR DESCRIPTION
**Pull request by sanjaykrishna1203**
### Functionality:

- Automated Quick notes to fetch title from description
- Stopped the fetch i the user manually modifies the title

### Solution:
Trimmed the Description and set to Title and stopped it if the user modifies the title

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:

- Click on CreateNew Note and type soemething in the description , chcek whether the title changes according to description.
- Change the title and check whether it remains same even after change in the description

### CODE REVIEW

**Review by Pendyala-thanvi — COMMENTED** [review]
This pr has changes from Pr #474 too. Please close the other pr.

**Comment by sanjaykrishna1203** [comment]
Done

**Review by viv9k — COMMENTED** [review]
approved need demo

File: src/app/services/applicationSettings/application-settings.service.ts [hunk 1]
```diff
@@ -51,7 +51,8 @@ export class ApplicationSettingsService {
 
   constructor(private userService: UserServiceService, private backendService: BackendService, private functions: AngularFireFunctions, private authService: AuthService) { }
 
-  getTeamDetails(teamId: string) {
+  getTeamDetails(teamId: string, allTeams?:boolean) {
+    if(this.team == undefined || this.team.TeamId != teamId || allTeams) {
```
Comment (viv9k) [inline src/app/services/applicationSettings/application-settings.service.ts:55]: What is the use of this change now, it's already solved by me.

File: src/app/body/view-organization-details/view-organization-details.component.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export class ViewOrganizationDetailsComponent implements OnInit {
   }
 
   getTeamDetails(teamId: string) {
-    this.applicationSettingsService.getTeamDetails(teamId).subscribe(data => {
+    this.applicationSettingsService.getTeamDetails(teamId, true).subscribe(data => {
```
Comment (viv9k) [inline src/app/body/view-organization-details/view-organization-details.component.ts:69]: No need for true etc

**Review by viv9k — CHANGES_REQUESTED** [review]
Please fix the required changes

File: src/app/services/applicationSettings/application-settings.service.ts (hunk 1 above)
Comment (sanjaykrishna1203) [reply src/app/services/applicationSettings/application-settings.service.ts]: Removed

File: src/app/body/view-organization-details/view-organization-details.component.ts (hunk 2 above)
Comment (sanjaykrishna1203) [reply src/app/body/view-organization-details/view-organization-details.component.ts]: Removed

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**